src/build.sh

```diff
@@ -31,6 +31,16 @@ die() {
   exit 1
 }
 
+## sanity check
+sanity_check () {
+  GSIV="$(gsi -v)"
+  case "$GSIV" in v4.9.[3-9]*" "20[2-9][0-9][0-1][0-9][0-3][0-9][0-2][0-9][0-5][0-9][0-9][0-9]" "*" \""*"\"") : ok ;;
```

Comment:
this will fail for versions earlier than 4.9.3 and later than 5.0; not sure we want to do that.

Reply:
also the error message will be very bad then.

Reply:
I could widen the version check to `v[4-9].[0-9].[0-9]`

Reply:
Widened.

Reply:
Looking by date is a mistake, recent versions use the git commit.
I think we should simply drop this check altogether, it doesn't add anything; comparing the version output between gsi and gsc is what we need.

Reply:
I'm not "looking by date", I'm just checking that a date is present, as is the case in the output, *after* the git commit information (which is covered by the `*`). I'm not checking the version anymore, either, just that it's in the format x.y.z where x>=4.

Reply:
This all seems redundant; why bother for that date?
Also, this code is fragile; once gambit v5.0.0 is released it's instabreak and requirement to change it.
I strongly recommend removing it altogether, I don't see what it adds.